### PR TITLE
[ Feature ] Add "Network Unavailable" alert dialog, which prompts user to open network preferences

### PR DESCRIPTION
**Pull request by bkonyi**
Not hooked up to anything yet, just creating the dialog in preparation for resolving #7.

Screenshot:

image

### CODE REVIEW

File: lib/src/ui/assets/theme.dart [hunk 1]
```diff
@@ -7,6 +7,7 @@ final ThemeData appTheme = ThemeData.light().copyWith(
   dialogTheme: DialogTheme(
     titleTextStyle: TextStyle(color: Colors.black),
     contentTextStyle: TextStyle(color: Colors.black),
+    shape: RoundedRectangleBorder(borderRadius: BorderRadius.circular(5)),
```
Comment (ferndot) [inline lib/src/ui/assets/theme.dart:10]: Have you tested this change with the temperature instructions dialog in step 2 of the checkup flow?

Comment (dnfield) [reply lib/src/ui/assets/theme.dart:10]: Should users still be able to work offline in this case, and store the data locally?

Comment (bkonyi) [reply lib/src/ui/assets/theme.dart:10]: We don't currently apply the `dialogTheme` to that dialog. Still looks the same to me after a quick look.

Comment (ferndot) [reply lib/src/ui/assets/theme.dart:10]: Ok 👍

**Review by ferndot — DISMISSED** [review]
This is looking really nice 😍!

**Comment by bkonyi** [comment]
> This is looking really nice 😍!

I'm glad you think so! I just fixed a merge conflict and it looks like it dismissed your review :( Mind reapproving?

**Comment by jonahwilliams** [comment]
nit: add a simple unit test

**Comment by ferndot** [comment]
@bkonyi do you want to add that unit test and then I will re-approve?

**Comment by bkonyi** [comment]
> @bkonyi do you want to add that unit test and then I will re-approve?

Sure, I can do that. @jonahwilliams what kind of test are we looking for? Something like:

- Navigate to the home page
- Tap the debug network icon
- Tap "Open Network Preferences" and confirm it opens the network page?

**Comment by bkonyi** [comment]
Landing a test for this is blocked by #84, so I think we can just go ahead and land this for now.

**Review by jonahwilliams — APPROVED** [review]
LGTM

**Pull request merged**